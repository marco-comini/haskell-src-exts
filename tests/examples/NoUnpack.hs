data UI = UI {-# NOUNPACK #-} !Int
