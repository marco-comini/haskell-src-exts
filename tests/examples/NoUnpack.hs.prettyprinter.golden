module Main (main) where

data UI = UI {-# NOUNPACK #-} !Int
